var/spack/repos/builtin/packages/wx/package.py

```diff
@@ -36,16 +36,20 @@ class Wx(Package):
        open-source and mature."""
     homepage = "http://www.wxwidgets.org/"
 
-    version('2.8.12', '2fa39da14bc06ea86fe902579fedc5b1',
-            url="https://sourceforge.net/projects/wxwindows/files/2.8.12/wxWidgets-2.8.12.tar.gz")
+    version('3.1.0', '2170839cfa9d9322e8ee8368b21a15a2497b4f11',
+            url="https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2")
+    version('3.0.2', '6461eab4428c0a8b9e41781b8787510484dea800',
+            url="https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.2/wxWidgets-3.0.2.tar.bz2")
     version('3.0.1', 'dad1f1cd9d4c370cbc22700dc492da31',
             url="https://sourceforge.net/projects/wxwindows/files/3.0.1/wxWidgets-3.0.1.tar.bz2")
+    version('2.8.12', '2fa39da14bc06ea86fe902579fedc5b1',
+            url="https://sourceforge.net/projects/wxwindows/files/2.8.12/wxWidgets-2.8.12.tar.gz")
```

Comment:
The best way to handle these changing URLs is to write a `url_for_version` function like in HDF5, but you don't have to.